src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -59,12 +59,12 @@ also encourage users to provide additional meta information extracted from the
 manufacturer specific data files in the sidecar JSON file. Other relevant files
 MAY be included alongside the original EEG data in `/sourcedata`.
 
-Note the RecordingType, which depends on whether the data stream on disk 
```

Comment:
@Remi-Gau I just rebased my changes on top of your "e.g." -> "for example" changes and noticed the trailing blank lines. Just as a hint for you: It may be nice (for any project you are doing) to configure your editor to automatically remove trailing whitespace.

Or is there a particular reason for having that?